README.md

```diff
@@ -1,981 +1,273 @@
-it unusable for running Cachito locally. Use the script from the `devel` branch instead.
-To facilitate this, set `CACHITO_COMPOSE_ENGINE` to the special value `podman-compose-auto`.
-which will instruct the Makefile to download and use the correct version of `podman-compose`.
-Be sure to pre-install the dependencies required by `podman-compose`, currently `PyYAML`.
-The script is available in `./tmp/podman_compose.py`. You may use this script to interact with
-the local deployment.
-
-```bash
-make run CACHITO_COMPOSE_ENGINE=podman-compose-auto
-```
+* **Auditable** - by generating a manifest of all the dependencies that go into your build.
 
-Verify in the browser at [http://localhost:8080/](http://localhost:8080/)
+The ability to achieve the goals depends on the hermeticity of the build process. Ideally, you should try to isolate the
+build from both the internet and the underlying host system to avoid implicit dependencies, irreproducible behavior and
+whole hosts of other issues. Cachi2 itself is not a hermetic build system. We suggest you take advantage of existing
+technologies - such as containers - to achieve isolation (see [usage](#usage)).
 
-Use curl to make requests:
+## Usage
 
-```bash
-# List all requests
-curl http://localhost:8080/api/v1/requests
+Let's show Cachi2 usage by building the glorious [fzf](https://github.com/junegunn/fzf) CLI tool hermetically. To follow
+along, clone the repository to your local disk.
 
-# Create a new request
-curl -X POST -H "Content-Type: application/json" http://localhost:8080/api/v1/requests -d \
-    '{
-        "repo": "https://github.com/release-engineering/retrodep.git",
-        "ref": "e1be527f39ec31323f0454f7d1422c6260b00580",
-        "pkg_managers": ["gomod"]
-      }'
-
-# Check the status of a request
-curl http://localhost:8080/api/v1/requests/1
-
-# Download the source archive for a completed request
-curl http://localhost:8080/api/v1/requests/1/download -o source.tar.gz
+```shell
+git clone https://github.com/junegunn/fzf --branch=0.34.0
 ```
 
-## Pre-built Container Images
-
-Cachito container images are automatically built when changes are merged. There are two images:
-an httpd based image with the Cachito API and a Celery worker image with the Cachito worker code.
-
-[![cachito-api](https://quay.io/repository/containerbuildsystem/cachito-api/status)](https://quay.io/repository/containerbuildsystem/cachito-api)
-  `quay.io/containerbuildsystem/cachito-api:latest`
-
-[![cachito-workers](https://quay.io/repository/containerbuildsystem/cachito-workers/status)](https://quay.io/repository/containerbuildsystem/cachito-workers)
-  `quay.io/containerbuildsystem/cachito-workers:latest`
+If you don't have the Cachi2 CLI installed locally, you can run it via a container image.
 
-## Prerequisites
+TODO: Not yet
 
-This is built to be used with Python 3.
+```shell
+alias cachi2='podman run --rm -ti -v "$PWD:$PWD:z" -w "$PWD" quay.io/containerbuildsystem/cachi2'
```

Comment:
@brunoapimentel are you planning to finish the Containerfile PR? If yes, I will test if this actually works. If not, I'll leave it here and hope that it does :smile:

Reply:
I just addressed the comments there, should be ready now.

Reply:
Tested and works (except that there's no `:latest` tag yet, so I used a specific one)